web/src/App.js

```diff
@@ -595,7 +601,7 @@ class App extends Component {
                   // theme="dark"
                   mode={(Setting.isMobile() && this.isStartPages()) ? "inline" : "horizontal"}
                   selectedKeys={[`${this.state.selectedMenuKey}`]}
-                  style={{lineHeight: "64px", width: "80%", position: "absolute", left: "145px"}}
+                  style={{lineHeight: "64px", width: "75%", position: "absolute", left: "145px"}}
```

Comment:
Is this still needed?

Reply:
It's not needed, now the width is calculated by the css function `calc()`.